### DISCUSSION TITLE
How to dowload extensions that are blocked.

### DISCUSSION THREAD

**Discussion by gengargaming1651** [3kh0/ext-remover#510] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I get the "Your admin has blocked" message when trying to download an extension. this is every extension but the ones that they allowed. Does anyone know a way around this or even a way to download extensions? Load unpacked does not work because of management btw.

**Comment by Aka-but-nice** [comment]
There's no way to bypass this block, I would never recommend using sh1mmer, but if you have to, sh1mmer would be your best bet.

**Reply by gengargaming1651** [reply]
I don't plan on unenrolling my Chromebook solely because of the consequences.

**Reply by fnfgo** [reply]
> I don't plan on unenrolling my Chromebook solely because of the consequences.

sh1mmer doesnt just unenrool your chromebook you know.

**Comment by obkduishinijf** [comment, marked as answer]
use sh1mmer if you're really that desperate. note sh1mmer doesn't just unenroll, there's more that you can do

**Reply by fnfgo** [reply]
> use sh1mmer if you're really that desperate. note sh1mmer doesn't just unenroll, there's more that you can do

bruh i  just fucking said that

**Reply by obkduishinijf** [reply]
> > use sh1mmer if you're really that desperate. note sh1mmer doesn't just unenroll, there's more that you can do
> 
> bruh i just fucking said that

"bruh i just fucking said that" 3rd time you've replied to me, bitching about me saying to use sh1mmer, clearly you're obsessed with me over us having the same thought. turn off your computer, and instead of playing games, play w/ some bitches.

**Reply by fnfgo** [reply]
> > > use sh1mmer if you're really that desperate. note sh1mmer doesn't just unenroll, there's more that you can do
> > 
> > 
> > bruh i just fucking said that
> 
> "bruh i just fucking said that" 3rd time you've replied to me, bitching about me saying to use sh1mmer, clearly you're obsessed with me over us having the same thought. turn off your computer, and instead of playing games, play w/ some bitches.

If i already said to use sh1mmer what part of your autistic brain thinks you should say the same thing.
My awensers were there before yours meaning youre just a copying idiot also youre probably in first grade so stop talking wiht your roblox girlfriend and go to bed its your nap time🥱

**Reply by Aka-but-nice** [reply]
Y'all stfu, y'all are acting like babies

**Reply by fnfgo** [reply]
> Y'all stfu, y'all are acting like babies

Need i remind you that hes in first grade
